src/azure-cli/azure/cli/command_modules/netappfiles/commands.py

```diff
@@ -126,6 +126,16 @@ def load_command_table(self, _):
                                  doc_string_source='azure.mgmt.netapp.models#VolumePatch',
                                  exception_handler=netappfiles_exception_handler)
 
+    with self.command_group('netappfiles volume replication', netappfiles_volumes_sdk) as g:
+        g.custom_command('approve', 'authorize_replication',
+                         client_factory=volumes_mgmt_client_factory,
+                         doc_string_source='azure.mgmt.netapp.models#Volume',
+                         exception_handler=netappfiles_exception_handler)
+        g.command('suspend', 'break_replication')
+        g.command('resume', 'resync_replication')
+        g.command('remove', 'delete_replication')
```

Comment:
what about changing to `show` command and provide confirmation kwargs?

Reply:
Unless I am misunderstanding I am not sure show is appropriate here this is a custom command that performs an action on approves/authorize the replication connection on the source volume.